src/components/ProgressIndicator/ProgressIndicatorItem.js

```diff
@@ -0,0 +1,75 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import Body from 'components/Body';
+import Link from 'components/Link';
+import {getBEMClassName} from 'utils';
+
+import CompletionMark from './CompletionMark';
+
+const ProgressIndicatorItem = ({
+  text,
+  href,
+  isActive,
+  isCompleted,
+  canNavigateTo,
+  isApplicable,
+  fixedText = null,
```

Comment:
the way this is used only as `fixedText || text` makes me think that there's no need at all for this prop - instead I'd just always pass `text`?

```jsx
<>
  <ProgressIndicatorItem text={step.formDefinition} />
  <ProgressIndicatorItem text="Overview" />
  <ProgressIndicatorItem text={<FormattedMessage description="..." defaultMessage="Confirmation" />} />
<>
```
are all equivalent.

the proptype would then be `text: PropTypes.node` which is "anything that can be rendered".

I'd also consider renaming `text` to `label` or perhaps even use `children`, since `text` seems to imply only `string` is allowed.

If you go the `children` route, you can do things like:

```jsx
<>
  <ProgressIndicatorItem children={step.formDefinition} />
  <ProgressIndicatorItem>{step.formDefinition}</ProgressIndicatorItem>
</>
```
 - these both render the same.